<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cg3::viewer::MeshManager</class>
 <widget class="QFrame" name="cg3::viewer::MeshManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QRadioButton" name="pointsRadioButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/points.png</normaloff>:/points.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="flatRadioButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/flat.png</normaloff>:/flat.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="smoothRadioButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/smooth.png</normaloff>:/smooth.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="wireframeCheckBox">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/wire.png</normaloff>:/wire.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
